logging/WFCORE-482-log4j2-support.adoc

```diff
@@ -0,0 +1,87 @@
+= Add log4j2 Support to WildFly
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+As log4j2 has risen in popularity and suggested as a replacement for log4j. There have been many requests to have support
+in WildFly for log4j2. With this feature users will be able to use the log4j2 API which will delegate to the JBoss Log
+Manager.
+
+NOTE: | This will be an implementation of the log4j2 API only. The core log manager for log4j2 will *not* be supported.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-482[WFCORE-482]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1352[EAP7-1352]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+* Any consumer of WildFly Core that uses the logging subsystem.
+
+== Requirements
+
+=== Hard Requirements
+
+* A user can use a log4j2 logger, `org.apache.logging.log4j.Logger`, which will pass the log message to the JBoss Log
+  Manager which is the log manager WildFly uses.
+* The log4j-api and implementation will be added to all deployments by default.
+* The log4j2 loggers created should honor the JBoss Log Manager's log context associated with the deployment.
+** This means if the deployment includes a logging configuration for a supported log manager the logger created should
```

Comment:
@jamezp same as above.... I'd some more info to understand this

Reply:
By supported log manager this means only a log4j 1.x or JBoss Log Manager configuration file. slf4j, commons-logging and jboss-logging are only logging facades so they do not support any specific configuration files.